CONTRIBUTING.md

```diff
@@ -82,6 +82,25 @@ The checksum can be updated automatically with the `xgensum` helper from the [xt
 
     $ xgensum -i <pkgname>
 
+### Adopting a template
+
+If a template is orphaned (maintained by `orphan@voidlinux.org`) or the current `maintainer` has not contributed in a long
+time (over a year), template maintainership can be adopted by someone else. To ensure a template gets the care it needs,
+template adopters should use the package in question and have a history of contributing changes to that template (updating
+the template several times, for example).
+
+It is best to adopt a template when making another change to it. When adopting the template, add your name or username
+and email to the `maintainer` field in the template, and mention the adoption in your commit message, for example:
+
+    libfoo: update to 1.2.3, adopt.
+
+### Orphaning a template
+
+If you no longer wish to maintain a template, you can remove yourself as maintainer by setting the `maintainer` field in
+the template to `Orphaned <orphan@voidlinux.org>`. The commit message should mention this, for example:
```

Comment:
Might also want to mention that a package can be orphaned without having to make another change to the template?